@uportal/esco-content-menu/src/components/ContentMenu.vue

```diff
@@ -83,14 +88,29 @@ export default {
     defaultOrgLogo: {type: String, required: true},
     userInfoPortletUrl: {type: String, default: ''},
     apiUrlOrgInfo: {type: String, default: ''},
+    favoritesPortletCardSize: {
+      validator: (value) =>
+        ['auto', 'large', 'medium', 'small', 'smaller'].includes(value),
+      default: 'auto',
+    },
+    gridPortletCardSize: {
+      validator: (value) =>
+        ['auto', 'large', 'medium', 'small', 'smaller'].includes(value),
```

Comment:
This function is being reused in several places.
Would it be helpful to have it be part of `sizeTools`?